console/src/routes.tsx

```diff
@@ -38,20 +38,22 @@ import aboutConnector from './components/Services/Settings/About/About';
 
 import { showErrorNotification } from './components/Services/Common/Notification';
 import { CLI_CONSOLE_MODE } from './constants';
-import UIKit from './components/UIKit/';
+import UIKit from './components/UIKit';
 import { Heading } from './components/UIKit/atoms';
+import { ReduxState, ReplaceRouterState } from './types';
 
-const routes = store => {
+
+const routes = (store: any) => {
   // load hasuractl migration status
-  const requireMigrationStatus = (nextState, replaceState, cb) => {
+  const requireMigrationStatus = (nextState: any, replaceState: any, cb: any) => {
```

Comment:
Not sure what types should be here. Definitely types of `any` should be replaced

Reply:
Suggestion: use `unknown` instead of `any`.

Reply:
Thanks, @kolharsam! Just updated!

Reply:
No, we can't delete `nextState` and `replaceState`, because `composeOnEnterHooks` function relies on it. 

Types there should be the one you added to `composeOnEnterHooks`. And store above should be typed as `ReduxState` (from `src/types.ts`).

Reply:
@beerose I tried to replace `any` with `ReduxState` type for `store`, but the code complains about "`dispatch` is not defined in `ReduxState`". Should I define `dispatch` in the `ReduxState` or the `store` should have have a different type?